randomkick.py

```diff
@@ -139,7 +142,19 @@ async def kick_user(delay, *, warn):
             await chosen.wait_save(delay)
         except asyncio.TimeoutError:
             try:
-                await bot.kick_participant(GROUP, chosen.id)
+                warn_message = await bot.get_message(GROUP, warn_message_id)
+                # message deleted or expired
+                if not warn_message:
+                    await bot.send_message(
+                        GROUP,
+                        f'Who the he*ck deleted my warning message? 3:<\n'
+                        'Guess I shouldn\'t kick <a href="tg://user?id={chosen.id}">{chosen.name}</a>'
+                        f' for being inactive…',
+                        parse_mode='html')
+                    # clear wait delay
+                    chosen.clicked_save()
```

Comment:
This section should be outside the try-except-else, otherwise it'll send the "user was kicked for being inactive" after sending the delete anger message